cf_units/__init__.py

```diff
@@ -398,13 +398,13 @@ def date2julian_day(date, calendar):
 
         >>> import cf_units
         >>> import datetime
-        >>> cf_units.date2julian_day(datetime.datetime(1970, 1, 1, 0, 0, 0),
+        >>> cf_units.date2julian_day(cftime.datetime(1970, 1, 1, 0, 0, 0),
         ...                          cf_units.CALENDAR_STANDARD)
         2440587.5...
 
     """
 
-    return cftime.JulianDayFromDate(date, calendar)
+    return date.toordinal(fractional=True)
```

Comment:
Note the use of the `fractional` keyword here means that the function will no longer work if a python `datetime.datetime` object is passed.  I note that the docstring specifies a `cftime.date` object should be passed, but the doctest previously used a `datetime.datetime`.  Without the `fractional` keyword, the doctest still failed on the `datetime.datetime`, because a very different number was returned.  Open to suggestions on the best thing to do here!